flocker_bb/builders/flocker.py

```diff
@@ -599,4 +691,19 @@ def getSchedulers():
             properties=[],
             builderNames=BUILDERS,
             ),
+        Triggerable(
+            name='trigger/homebrew-create',
```

Comment:
I've been naming the triggers after the triggering event, not what it should trigger.